src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -289,6 +289,14 @@ def __publish_new_nodes(
 
         self.dal.publish_nodes(updated_nodes)
 
+    def __safe_delete_job(self, job_key):
+        try:
+            self.__jobs_cache[job_key].deleted = True
```

Comment:
I think we don't need this any more.
We changed the deleted field to True, because we were doing soft delete. 
So we were updating the DB with deleted=true
But now we're using hard delete

Reply:
todo: add comment why this is important